src/Tribe/Aggregator/Service.php

```diff
@@ -172,7 +172,14 @@ public function get( $endpoint, $data = array() ) {
 			return $response;
 		}
 
-		if ( isset( $response->data ) && isset( $response->data->status ) && '404' === $response->data->status ) {
+		if (
+			(
+				isset( $response->data )
+				&& isset( $response->data->status )
+				&& '404' === $response->data->status
+			)
+		     || '200' != wp_remote_retrieve_response_code( $response )
```

Comment:
For `403`:

```php
return new WP_Error( 'core:aggregator:request-limit', esc_html__( 'The Event Aggregator service has blocked this request because it came too soon after a previous request.', 'the-events-calendar' ) );
```

For `! 200`:

```php
return new WP_Error( 'core:aggregator:invalid-service-response', esc_html__( 'There may be an issue with the Event Aggregator server. Please try your import again later.', 'the-events-calendar' ) );
```